### DISCUSSION TITLE
Random Projects {WIP}

### DISCUSSION THREAD

**Discussion by JRL101** [tsunamayo/Starship-EVO#5002] [General]
Feel free to show off incomplete or interesting designs. In game screenshots only.
![image](https://user-images.githubusercontent.com/33255536/202871531-2aa7fc89-141c-434a-a822-0936e311f702.png)
![image](https://user-images.githubusercontent.com/33255536/202871543-0ab209ff-0f94-43c1-9dbf-7bb95e1ae37b.png)
![image](https://user-images.githubusercontent.com/33255536/202871591-22d2acdf-0cdc-4e20-ac5c-b5dcd9c27fc7.png)
![image](https://user-images.githubusercontent.com/33255536/202871604-88570502-e7cd-4f77-9445-e9bdb941c8ff.png)
![image](https://user-images.githubusercontent.com/33255536/202871621-13eb2a3e-b741-43a5-9d06-32099c6f1382.png)
![image](https://user-images.githubusercontent.com/33255536/202871636-ac963a71-be11-40d4-9e77-c2bbba699bcd.png)
![image](https://user-images.githubusercontent.com/33255536/202871662-4af27350-2257-4b35-b652-c5e7e126d4da.png)
^ this is one of those projects i got mentally drained trying to figure out a shape for the hull. One of the simpler cockpits i've designed. 
real rusty cramped feeling

**Comment by JRL101** [comment]
![image](https://user-images.githubusercontent.com/33255536/202872117-c78f91bd-8266-499d-80e6-ba81ed1a9a4d.png)
![image](https://user-images.githubusercontent.com/33255536/202872137-40871a6f-98bd-4d4f-be4b-d4672cd2fcd7.png)
![image](https://user-images.githubusercontent.com/33255536/202872157-6612f070-e9b1-4338-879c-2bb428200219.png)
![image](https://user-images.githubusercontent.com/33255536/202872165-3cf57bc6-190b-4b50-a934-44f8efdfdd9a.png)
![image](https://user-images.githubusercontent.com/33255536/202872177-df30585d-d53c-4849-b3cf-1da8f5905fed.png)
![image](https://user-images.githubusercontent.com/33255536/202872185-34bfdfaf-42aa-40ee-b87e-167a8d8c96cd.png)
![image](https://user-images.githubusercontent.com/33255536/202872195-8d07835e-76e3-48ce-b717-1a1721647c29.png)

**Comment by Briaireous** [comment]
Lol fam, we got discord for this
